src/commands.rs

```diff
@@ -1539,9 +1539,7 @@ fn cmd_status(
             formatter.with_label("conflict", |formatter| {
                 writeln!(formatter, "There are unresolved conflicts at these paths:")
             })?;
-            for (path, _) in conflicts {
-                writeln!(formatter, "{}", &workspace_command.format_file_path(&path))?;
-            }
+            print_conflicted_paths(&conflicts, &tree, formatter, &workspace_command)?
```

Comment:
It would be nice if the paths were indented just like the conflicted branches above are, but that can be fixed later.

Reply:
The simplest thing would be to have a `formatter.with_indent` function. I also imagine `jj log` has to do something even fancier to print the commit descriptions to the right of the graph.